com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1117,11 +1129,13 @@ public override void Initialize(NetworkManager networkManager = null)
             // account for the overhead of its length when we store it in the send queue.
             var fragmentationCapacity = m_MaxPayloadSize + BatchedSendQueue.PerMessageOverhead;
 
-            m_NetworkSettings
-                .WithFragmentationStageParameters(payloadCapacity: fragmentationCapacity)
+            m_NetworkSettings.WithFragmentationStageParameters(payloadCapacity: fragmentationCapacity)
+#if !UTP_TRANSPORT_2_0_ABOVE
                 .WithBaselibNetworkInterfaceParameters(
                     receiveQueueCapacity: m_MaxPacketQueueSize,
-                    sendQueueCapacity: m_MaxPacketQueueSize);
+                    sendQueueCapacity: m_MaxPacketQueueSize)
+#endif
+                ;
```

Comment:
This could perhaps be a bit simpler if written this way:

```csharp
m_NetworkSettings.WithFragmentationStageParameters(...);
#if !UTP_TRANSPORT_2_0_ABOVE
m_NetworkSettings.WithBaselibNetworkInterfaceParameters(...);
#endif
```

Reply:
Ok, I didn't know enough of the internals to be sure it did the exact same. I'll change it. Thanks!